### ISSUE TITLE
Don't compute variables for years we haven't implemented yet

### ISSUE THREAD

**Issue by MaxGhenis** [PolicyEngine/openfisca-us#934]
Currently you can specify years we haven't implemented for many tax and benefit programs. I think the simplest way to avoid this is to replace `def formula` with `def formula_2021` or whatever the first year we have that's vetted. It's also possible to return `NaN`s.

Thanks @martinholmer for the tip.

**Comment by nikhilwoodruff** [comment]
Yes, I'd favour setting the default value to np.nan (with a constant named `NULL` or something), and using dated formulas.
